xclim/core/indicator.py

```diff
@@ -62,7 +65,7 @@
         allowed_periods: [<list>, <of>, <allowed>, <periods>]
 
         # Compute function
-        compute: <function name>  # Referring to a function in the supplied `Indices` module, xclim.indices.generic or xclim.indices
+        compute: <function name>  # Referring to a function in `Indices` module (xclim.indices.generic or xclim.indices)
         input:  # When "compute" is a generic function, this is a mapping from argument name to the expected variable.
```

Comment:
if Ruff is responsible for this removal, it is being quite opiniated, no?

Reply:
That is stylistic IMO

Reply:
Nope, again, this was my doing.